lib/compress/index.js

```diff
@@ -145,7 +145,8 @@ import {
 
     _INLINE,
     _NOINLINE,
-    _PURE
+    _PURE,
+    AST_Case
```

Comment:
Could you place this import in line with other AST_* imports? There's some ordering here, since this is such a large import statement.